specification/columns/capacityreservationid.md

```diff
@@ -0,0 +1,31 @@
+# Capacity Reservation ID
+
+A Capacity Reservation ID is the identifier assigned to a [*capacity reservation*](#glossary:capacity-reservation) by the provider. Capacity Reservation ID is commonly used for scenarios like chargeback for capacity reservation usage.
+
+The CapacityReservationId column MUST be present in the billing data when the provider supports capacity reservations. This column MUST be of type String and MUST NOT contain null values when a charge is related to a capacity reservation. When a charge is not associated with a capacity reservation, the column MUST be null. CapacityReservationId MUST be unique within the provider.
```

Comment:
Wouldn't this be a "SHOULD" since AWS will not provide any ID values?

Reply:
good question; I pondered the same. Let's see how the conversation goes on Friday.

Reply:
Lastly, needs to include normative text around corrections.  Something like ... `The value MAY be negative in cases where ChargeClass is "Correction`.

Reply:
I went ahead and made all of these changes.